jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPluginActivity.kt

```diff
@@ -18,7 +19,7 @@ import org.jitsi.meet.sdk.JitsiMeetConferenceOptions
 /**
  * Activity extending JitsiMeetActivity in order to override the conference events
  */
-class JitsiMeetPluginActivity : JitsiMeetActivity() {
+class JitsiMeetPluginActivity<Activity> : JitsiMeetActivity() {
```

Comment:
I don't quite understand this. Won't this just add a new generic variable? Do we use that variable anywhere?

Reply:
i don't understand what do you mean here. it just a return type here. v3 forces you to put retun type.

Reply:
How does a class have return type? Isn't that a [generic type variable](https://kotlinlang.org/docs/generics.html)?

Reply:
i mean it specifies the type of the class itself. it doesn't have the need to be generic. the class is extendeing formt he jitsimeetactivity which is extened further from the activity android. so for that we need to explicitly defined it's type otherwise it will not compile
Generic doesn't work